lib/matplotlib/axes/_axes.py

```diff
@@ -794,6 +794,69 @@ def axvline(self, x=0, ymin=0, ymax=1, **kwargs):
         return l
 
     @docstring.dedent_interpd
+    def axline(self, slope=1, intercept=0, **kwargs):
+        """
+        Add an infinite line across the axes.
+
+        Parameters
+        ----------
+        slope : scalar, optional, default: 1
```

Comment:
Is there a reason this and intercept are optional? I would say the user should be forced to give them.